src/biome/data/sources/readers.py

```diff
@@ -64,13 +70,18 @@ def json_engine(*args, flatten: bool = False, **kwargs) -> pd.DataFrame:
         df = pd.read_json(*args, **kwargs)
         return flatten_dataframe(df) if flatten else df
 
-    ddf = dd.read_json(path, flatten=flatten, engine=json_engine, **params)
-    ddf["path"] = path
+    path_list = _get_file_paths(path)
```

Comment:
Same comment for `read_json`

https://docs.dask.org/en/latest/dataframe-api.html#dask.dataframe.read_json